vpr/src/place/compressed_grid.cpp

```diff
@@ -124,3 +124,13 @@ int grid_to_compressed(const std::vector<int>& coords, int point) {
 
     return std::distance(coords.begin(), itr);
 }
+
+//find the nearest location in the compressed grid
+//Useful when the point is of a different type from coords
+int grid_to_compressed_approx(const std::vector<int>& coords, int point) {
+    auto itr = std::lower_bound(coords.begin(), coords.end(), point);
+    //VTR_ASSERT(*itr == point);
```

Comment:
Why is this commented?

Reply:
The point might be of a different type than the coords. Hence, we will never find the exact point in the coords. Instead, we search for the nearest location to this point in its compressed grid.